make.cmd

```diff
@@ -0,0 +1,8 @@
+@echo off
+
+FOR /F "tokens=1,2 delims==" %%A IN (mod.config) DO (set %%A=%%B)
+
+set MOD_SEARCH_PATHS=./mods,%~dp0mods
+
+:start
```

Comment:
The environment loading yes, but not the label, I wonder where that sneaked in again...